App.js

```diff
@@ -59,7 +59,11 @@ class App extends Component {
 
     persistStore(
       configureStore,
-      { storage: AsyncStorage, transforms: [encryptor] },
+      {
+        storage: AsyncStorage,
+        transforms: [encryptor],
+        blacklist: ['user'],
```

Comment:
We should probably `blacklist` more than just _user_ right? Something we need to discuss once we rearchitect our logic and state

Reply:
Yes, it is probably possible to supplement it. Need to discuss.